### PR TITLE
Move to SuperInvoke

### LINKED ISSUE #61: Move to SuperInvoke
Fixes #60 
Tracker Ultz/SuperTools#1

### PR DESCRIPTION
**Pull request by Perksey**
# Summary of the PR
Removes AdvancedDLSupport and replaces it with SuperInvoke - Ultz's homegrown MIT-licensed solution for native interoperation.

# Related issues, Discord discussions, or proposals
Closes #61 

# Further Comments

### CODE REVIEW

File: build/props/common.props [hunk 1]
```diff
@@ -12,4 +12,7 @@
         <Version>1.0.0-preview4$(VersionSuffix)</Version>
         <Description>Silk.NET is a high-speed, advanced library, providing bindings to popular low-level APIs such as OpenGL, OpenAL, and Vulkan.</Description>
     </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="Ultz.SuperInvoke" Version="1.0.0-preview20200202.3" /> <!-- TODO remove this! -->
```
Comment (devvoid) [inline build/props/common.props]: Is this todo meant to be here?

Comment (Perksey) [reply build/props/common.props]: Yes because at the moment it's referencing the experimental feed as there isn't a public build of SuperInvoke yet. SuperInvoke will be pushed to the public feed just before Preview 4 releases, and this will be removed from common.props and into the appropriate csprojs. This is just so I could quickly test it.

TL;DR will be rectified in a future PR

Comment (devvoid) [reply build/props/common.props]: Gotcha
